testhelpers/docker_registry.go

```diff
@@ -182,6 +193,39 @@ func (r *DockerRegistry) EncodedLabeledAuth() string {
 	return base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"username":"%s","password":"%s"}`, r.username, r.password)))
 }
 
+// SetPrivilegesToImage sets the given image name with the provided ImagePrivileges if ImagePrivileges was enabled.
+// For example SetPrivilegesToImage("my-image", NewImagePrivileges(Readable, Writable)) will save "my-image" as a
+// readable and writable image into the registry.
+func (r *DockerRegistry) setPrivilegesToImage(imageName string, privilege ImagePrivileges) {
+	if r.isImagePrivilegesEnable() {
+		r.imagePrivileges[imageName] = privilege
+	}
+}
+
+// MakeReadOnly set the given image name to be readable when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeReadOnly(imageName string) {
+	r.setPrivilegesToImage(imageName, NewImagePrivileges(Readable))
+}
+
+// MakeWriteOnly set the given image name to be writable when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeWriteOnly(imageName string) {
+	r.setPrivilegesToImage(imageName, NewImagePrivileges(Writable))
+}
+
+// MakeReadWrite set the given image name to be readable and writable when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeReadWrite(imageName string) {
+	r.setPrivilegesToImage(imageName, NewImagePrivileges(Readable, Writable))
+}
+
+// MakeInaccessible set the given image name to do not have any access when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeInaccessible(imageName string) {
+	r.setPrivilegesToImage(imageName, NewImagePrivileges())
+}
+
+func (r *DockerRegistry) isImagePrivilegesEnable() bool {
```

Comment:
I think that it should be: `isImagePrivilegesEnabled`

Reply:
I like `imagePrivilegesEnabled`

Reply:
Renamed to `imagePrivilegesEnabled` in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/c59c94c0966e2826f3e1ce6b48983be810952fee)